### PR TITLE
fix: fix console plugin's OOM problem

### PR DESCRIPTION
**Pull request by PeterChen1997**

### CODE REVIEW

**Comment by Yuyz0112** [comment]
LGTM

maybe @Mark-Fenng can do a review either?

File: packages/rrweb/src/plugins/console/record/stringify.ts [hunk 1]
```diff
@@ -48,6 +48,26 @@ function pathToSelector(node: HTMLElement): string | '' {
   return path;
 }
 
+/**
+ * judge the object's depth
+ */
+function isObjTooDeep(obj: any, limit: number): boolean {
```
Comment (Mark-Fenng) [inline packages/rrweb/src/plugins/console/record/stringify.ts:54]: This solution can solve the issue well but I prefer the way to judge depth in the stringify function.

**Comment by PeterChen1997** [comment]
@Mark-Fenng @Yuyz0112 fixed~

**Review by Mark-Fenng — APPROVED** [review]
This is really great now!

**Pull request merged**